recipes/mqc/run_test.py

```diff
@@ -0,0 +1,22 @@
+#Test Python modules installation
+import traceback
```

Comment:
You can put this into the mata.yaml as well: https://conda.io/docs/building/meta-yaml.html#python-imports